Dockerfile

```diff
@@ -22,7 +22,6 @@ RUN yum  -y update && \
                    which \
                    file \
                    unzip \
-                   lsof \
```

Comment:
why remove `lsof` here? if this was to prevent a double install ask, then `file`, `wget`, etc should have also been removed.